x-pack/plugins/observability_solution/infra/public/alerting/inventory/components/expression.tsx

```diff
@@ -120,15 +120,12 @@ export const defaultExpression = {
 
 export const Expressions: React.FC<Props> = (props) => {
   const { setRuleParams, ruleParams, errors, metadata } = props;
-  const { source, createDerivedIndexPattern } = useSourceContext();
+  const { source } = useSourceContext();
```

Comment:
how is this being used here?

Reply:
yes. we need the `sourceId`. Should we create a ticket to stop using this attribute? I don't really see the point of having it in the current solution.

Reply:
oops, missed that

Reply:
@elastic/obs-ux-management-team we might want to log the ticket Carlos mentions here and get rid of the source entirely if it's no longer used/needed, in favor of data views